app/code/community/Bolt/Boltpay/controllers/ShippingController.php

```diff
@@ -128,7 +128,6 @@ public function indexAction()
                     422,
                     array('status' => 'failure','error' => $addressErrorDetails)
```

Comment:
Great insight!
To preserve the log, if important at all, perhaps we could do this
```suggestion
                    422,
                    array('status' => 'failure','error' => $addressErrorDetails)	                    array('status' => 'failure','error' => $addressErrorDetails),
                   call_user_func(function(){benchmark('Sending shipping address failure to Bolt');return true;})
```